config/flipt.schema.json

```diff
@@ -1219,6 +1219,10 @@
                 "file": {
                   "type": "string",
                   "default": ""
+                },
+                "encoding": {
+                  "type": "string",
+                  "enum": ["json", "console", ""]
```

Comment:
the `""` is required so that by default it inherits the same encoding as `log.encoding`, although i guess we could make an explicit `inherit` option?